src/auth/session/storage/__tests__/mongodb.test.ts

```diff
@@ -0,0 +1,54 @@
+import * as child_process from 'child_process';
+import {promisify} from 'util';
+
+import * as mongodb from 'mongodb';
+
+import {MongoDBSessionStorage} from '../mongodb';
+
+import {batteryOfTests} from './battery-of-tests';
+import {poll} from './utils';
+
+const exec = promisify(child_process.exec);
+
+const dbURL = new URL('mongodb://shopify:passify@localhost');
+const dbName = 'shopitest';
+
+// SORRY NOT SORRY
+jest.setTimeout(20000);
+
+describe('MongoDBSessionStorage', () => {
+  let storage: MongoDBSessionStorage;
+  let containerId: string;
+  beforeAll(async () => {
```

Comment:
I wonder if it's worth extracting this into a utils function that takes in the `docker` command and a `poll` callback?

Reply:
Yeah, I may do that later. For now I didn’t want to hide too much of the magic.